lib/modules/manager/bazelisk/index.ts

```diff
@@ -0,0 +1,12 @@
+import { GithubReleasesDatasource } from '../../datasource/github-releases';
+import * as semverVersioning from '../../versioning/semver';
+
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  fileMatch: ['(^|/)\\.bazelversion$'],
+  versioning: semverVersioning.id,
```

Comment:
@rarkins i think we don't need this, as it's datasource default? 🤔

Reply:
Correct. It can be removed

Reply:
done